src/NHibernate/Impl/SessionImpl.cs

```diff
@@ -521,7 +531,7 @@ IList Find(string query, object[] values, IType[] types)
 			}
 		}
 
-		public override void CloseSessionFromDistributedTransaction()
+		public override void CloseSessionFromSystemTransaction()
```

Comment:
All "distributed" and "dtc" naming changed when it was not corresponding to something actually always distributed, but only to system transaction, which are not forcibly distributed. I think that bad naming was causing unneeded confusion.